<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changesales</class>
 <widget class="QDialog" name="changesales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改经办人信息</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>-10</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;方正舒体&quot;;</string>
   </property>
   <property name="text">
    <string>修改经办人信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>获取信息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>331</y>
     <width>171</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>331</y>
     <width>54</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>112</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>编号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_num"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>166</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_name"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>220</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>男</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">sexGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>女</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">sexGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>274</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>电话：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_phone"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sexGroup"/>
 </buttongroups>
</ui>
